--- spice-0.8.1/configure.ac.orig	2011-04-20 10:48:33.000000000 +0200
+++ spice-0.8.1/configure.ac	2011-07-16 12:18:43.082766454 +0200
@@ -138,7 +138,7 @@
    )
 AC_SUBST(LIBRT)
 
-SPICE_NONPKGCONFIG_LIBS+=" -pthread $LIBM $LIBRT"
+SPICE_NONPKGCONFIG_LIBS="$SPICE_NONPKGCONFIG_LIBS -pthread $LIBM $LIBRT"
 
 dnl The client needs a yield function
 AC_MSG_CHECKING(for posix yield function)
@@ -150,7 +150,7 @@
 		[$yield_func()],
 		[posix_yield_func="$yield_func"
 		break])
-    CPPFLAGS="spice_save_CPPFLAGS"
+    CPPFLAGS="$spice_save_CPPFLAGS"
 done
 if test x"$posix_yield_func" = xnone; then
     AC_MSG_ERROR([No posix yield function found])
@@ -167,13 +167,13 @@
     [
         AC_SUBST(CEGUI06_CFLAGS)
         AC_SUBST(CEGUI06_LIBS)
-        CEGUI06_CFLAGS+="-DUSE_GUI"
+        CEGUI06_CFLAGS="$CEGUI06_CFLAGS -DUSE_GUI"
     ],
     [
         PKG_CHECK_MODULES(CEGUI, CEGUI >= 0.6.0 CEGUI < 0.7.0)
         AC_SUBST(CEGUI_CFLAGS)
         AC_SUBST(CEGUI_LIBS)
-        CEGUI_CFLAGS+="-DUSE_GUI"
+        CEGUI_CFLAGS="$CEGUI_CFLAGS -DUSE_GUI"
     ])
 fi
 
@@ -181,7 +181,7 @@
 	PKG_CHECK_MODULES(SLIRP, slirp)
 	AC_SUBST(SLIRP_CFLAGS)
 	AC_SUBST(SLIRP_LIBS)
-	SPICE_REQUIRES+=" slirp"
+	SPICE_REQUIRES="$SPICE_REQUIRES slirp"
 	AC_DEFINE([HAVE_SLIRP], [], [Define if we have slirp])
 fi
 
@@ -197,13 +197,13 @@
 PKG_CHECK_MODULES(PIXMAN, pixman-1 >= 0.17.7)
 AC_SUBST(PIXMAN_CFLAGS)
 AC_SUBST(PIXMAN_LIBS)
-SPICE_REQUIRES+=" pixman-1 >= 0.17.7"
+SPICE_REQUIRES="$SPICE_REQUIRES pixman-1 >= 0.17.7"
 
 PKG_CHECK_MODULES(CELT051, celt051 >= 0.5.1.1)
 AC_SUBST(CELT051_CFLAGS)
 AC_SUBST(CELT051_LIBS)
 AC_SUBST(CELT051_LIBDIR)
-SPICE_REQUIRES+=" celt051 >= 0.5.1.1"
+SPICE_REQUIRES="$SPICE_REQUIRES celt051 >= 0.5.1.1"
 
 PKG_CHECK_MODULES(ALSA, alsa)
 AC_SUBST(ALSA_CFLAGS)
@@ -212,14 +212,14 @@
 PKG_CHECK_MODULES(SSL, openssl)
 AC_SUBST(SSL_CFLAGS)
 AC_SUBST(SSL_LIBS)
-SPICE_REQUIRES+=" openssl"
+SPICE_REQUIRES="$SPICE_REQUIRES openssl"
 
 # These are commented out because the gl libraries on RHEL 5 do not have pkgconfig files
 #
 # PKG_CHECK_MODULES(GL, gl glu)
 # AC_SUBST(GL_CFLAGS)
 # AC_SUBST(GL_LIBS)
-# SPICE_REQUIRES+=" gl glu"
+# SPICE_REQUIRES="$SPICE_REQUIRES gl glu"
 
 if test "x$have_opengl" = "xyes"; then
    AC_CHECK_LIB(GL, glBlendFunc, GL_LIBS="$GL_LIBS -lGL", have_opengl=no)
@@ -233,7 +233,7 @@
 
 AC_SUBST(GL_CFLAGS)
 AC_SUBST(GL_LIBS)
-SPICE_NONPKGCONFIG_LIBS+=" $GL_LIBS"
+SPICE_NONPKGCONFIG_LIBS="$SPICE_NONPKGCONFIG_LIBS $GL_LIBS"
 
 PKG_CHECK_MODULES(XRANDR, xrandr)
 PKG_CHECK_MODULES(XFIXES, xfixes)
