--- spice-0.10.0/configure.ac.orig	2011-11-10 16:02:06.000000000 +0100
+++ spice-0.10.0/configure.ac	2011-11-13 08:00:42.323766723 +0100
@@ -145,9 +145,9 @@
 AC_SUBST(LIBRT)
 
 if test "$red_target" = "windows"; then
-	SPICE_NONPKGCONFIG_LIBS+=" -lpthreadGC2 -lversion -lmsimg32 $LIBM"
+	SPICE_NONPKGCONFIG_LIBS="$SPICE_NONPKGCONFIG_LIBS -lpthreadGC2 -lversion -lmsimg32 $LIBM"
 else
-	SPICE_NONPKGCONFIG_LIBS+=" -pthread $LIBM $LIBRT"
+	SPICE_NONPKGCONFIG_LIBS="$SPICE_NONPKGCONFIG_LIBS -pthread $LIBM $LIBRT"
 fi
 
 dnl The client needs a yield function
@@ -191,7 +191,7 @@
 	PKG_CHECK_MODULES(SLIRP, slirp)
 	AC_SUBST(SLIRP_CFLAGS)
 	AC_SUBST(SLIRP_LIBS)
-	SPICE_REQUIRES+=" slirp"
+	SPICE_REQUIRES="$SPICE_REQUIRES slirp"
 	AC_DEFINE([HAVE_SLIRP], [], [Define if we have slirp])
 fi
 
@@ -207,13 +207,13 @@
 PKG_CHECK_MODULES(PIXMAN, pixman-1 >= 0.17.7)
 AC_SUBST(PIXMAN_CFLAGS)
 AC_SUBST(PIXMAN_LIBS)
-SPICE_REQUIRES+=" pixman-1 >= 0.17.7"
+SPICE_REQUIRES="$SPICE_REQUIRES pixman-1 >= 0.17.7"
 
 PKG_CHECK_MODULES(CELT051, celt051 >= 0.5.1.1)
 AC_SUBST(CELT051_CFLAGS)
 AC_SUBST(CELT051_LIBS)
 AC_SUBST(CELT051_LIBDIR)
-SPICE_REQUIRES+=" celt051 >= 0.5.1.1"
+SPICE_REQUIRES="$SPICE_REQUIRES celt051 >= 0.5.1.1"
 
 if test ! -e client/generated_marshallers.cpp; then
 AC_MSG_CHECKING([for pyparsing python module])
@@ -234,14 +234,14 @@
 PKG_CHECK_MODULES(SSL, openssl)
 AC_SUBST(SSL_CFLAGS)
 AC_SUBST(SSL_LIBS)
-SPICE_REQUIRES+=" openssl"
+SPICE_REQUIRES="$SPICE_REQUIRES openssl"
 
 # These are commented out because the gl libraries on RHEL 5 do not have pkgconfig files
 #
 # PKG_CHECK_MODULES(GL, gl glu)
 # AC_SUBST(GL_CFLAGS)
 # AC_SUBST(GL_LIBS)
-# SPICE_REQUIRES+=" gl glu"
+# SPICE_REQUIRES="$SPICE_REQUIRES gl glu"
 
 if test "x$enable_opengl" = "xyes"; then
    AC_CHECK_LIB(GL, glBlendFunc, GL_LIBS="$GL_LIBS -lGL", enable_opengl=no)
@@ -256,7 +256,7 @@
 
 AC_SUBST(GL_CFLAGS)
 AC_SUBST(GL_LIBS)
-SPICE_NONPKGCONFIG_LIBS+=" $GL_LIBS"
+SPICE_NONPKGCONFIG_LIBS="$SPICE_NONPKGCONFIG_LIBS $GL_LIBS"
 
 if test "$red_target" = "x11"; then
 	PKG_CHECK_MODULES(XRANDR, xrandr)
