--- spice-0.10.0/configure.ac.orig	2011-11-13 08:37:42.000000000 +0100
+++ spice-0.10.0/configure.ac	2011-11-13 08:38:30.260509328 +0100
@@ -246,6 +246,7 @@
 if test "x$enable_opengl" = "xyes"; then
    AC_CHECK_LIB(GL, glBlendFunc, GL_LIBS="$GL_LIBS -lGL", enable_opengl=no)
    AC_CHECK_LIB(GLU, gluSphere, GL_LIBS="$GL_LIBS -lGLU", enable_opengl=no)
+   AC_CHECK_LIB(X11, XOpenDisplay, GL_LIBS="$GL_LIBS -lX11", have_opengl=no)
    AC_DEFINE([USE_OPENGL], [1], [Define to build with OpenGL support])
    AC_DEFINE([GL_GLEXT_PROTOTYPES], [], [Enable GLExt prototypes])
 
