diff -urNp -x '*.orig' spice-0.14.3.org/configure.ac spice-0.14.3/configure.ac
--- spice-0.14.3.org/configure.ac	2020-02-27 12:26:12.000000000 +0100
+++ spice-0.14.3/configure.ac	2021-10-03 22:06:48.560189277 +0200
@@ -31,7 +31,7 @@ AC_SUBST(SPICE_SERVER_VERSION)
 
 AC_CONFIG_MACRO_DIR([m4])
 m4_include([subprojects/spice-common/m4/spice-deps.m4])
-AM_CONFIG_HEADER([config.h])
+AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_AUX_DIR(.)
 
 AM_INIT_AUTOMAKE([dist-bzip2 no-dist-gzip subdir-objects foreign])
