--- spice-0.12.5/configure.ac.orig	2014-05-19 14:20:53.000000000 +0200
+++ spice-0.12.5/configure.ac	2014-05-23 19:18:10.227726292 +0200
@@ -163,9 +163,9 @@
 AC_SUBST(LIBRT)
 
 if test "$red_target" = "windows"; then
-	SPICE_NONPKGCONFIG_LIBS+=" -lpthreadGC2 -lversion -lmsimg32 $LIBM"
+	SPICE_NONPKGCONFIG_LIBS="$SPICE_NONPKGCONFIG_LIBS -lpthreadGC2 -lversion -lmsimg32 $LIBM"
 else
-	SPICE_NONPKGCONFIG_LIBS+=" -pthread $LIBM $LIBRT"
+	SPICE_NONPKGCONFIG_LIBS="$SPICE_NONPKGCONFIG_LIBS -pthread $LIBM $LIBRT"
 fi
 
 dnl The client needs a yield function
@@ -211,24 +211,24 @@
     SMARTCARD_CFLAGS="$CAC_CARD_CFLAGS"
     AC_SUBST(SMARTCARD_LIBS)
     AC_SUBST(SMARTCARD_CFLAGS)
-    SPICE_REQUIRES+=" libcacard >= 0.1.2"
+    SPICE_REQUIRES="$SPICE_REQUIRES libcacard >= 0.1.2"
 fi
 
 
 PKG_CHECK_MODULES([GLIB2], [glib-2.0 >= 2.22])
-SPICE_REQUIRES+=" glib-2.0 >= 2.22"
+SPICE_REQUIRES="$SPICE_REQUIRES glib-2.0 >= 2.22"
 
 PKG_CHECK_MODULES(PIXMAN, pixman-1 >= 0.17.7)
 AC_SUBST(PIXMAN_CFLAGS)
 AC_SUBST(PIXMAN_LIBS)
-SPICE_REQUIRES+=" pixman-1 >= 0.17.7"
+SPICE_REQUIRES="$SPICE_REQUIRES pixman-1 >= 0.17.7"
 
 AC_ARG_ENABLE(celt051,
     [  --disable-celt051       Disable celt051 audio codec (enabled by default)],,
         [enable_celt051="yes"])
 
 if test "x$enable_celt051" = "xyes"; then
-    PKG_CHECK_MODULES(CELT051, celt051 >= 0.5.1.1, SPICE_REQUIRES+=" celt051 >= 0.5.1.1")
+    PKG_CHECK_MODULES(CELT051, celt051 >= 0.5.1.1, SPICE_REQUIRES="$SPICE_REQUIRES celt051 >= 0.5.1.1")
 fi
 
 if test ! -e client/generated_marshallers.cpp; then
@@ -244,14 +244,14 @@
 PKG_CHECK_MODULES(SSL, openssl)
 AC_SUBST(SSL_CFLAGS)
 AC_SUBST(SSL_LIBS)
-SPICE_REQUIRES+=" openssl"
+SPICE_REQUIRES="$SPICE_REQUIRES openssl"
 
 # These are commented out because the gl libraries on RHEL 5 do not have pkgconfig files
 #
 # PKG_CHECK_MODULES(GL, gl glu)
 # AC_SUBST(GL_CFLAGS)
 # AC_SUBST(GL_LIBS)
-# SPICE_REQUIRES+=" gl glu"
+# SPICE_REQUIRES="$SPICE_REQUIRES gl glu"
 
 if test "x$enable_opengl" = "xyes"; then
    AC_CHECK_LIB(GL, glBlendFunc, GL_LIBS="$GL_LIBS -lGL", enable_opengl=no)
@@ -266,7 +266,7 @@
 
 AC_SUBST(GL_CFLAGS)
 AC_SUBST(GL_LIBS)
-SPICE_NONPKGCONFIG_LIBS+=" $GL_LIBS"
+SPICE_NONPKGCONFIG_LIBS="$SPICE_NONPKGCONFIG_LIBS $GL_LIBS"
 
 if test "x$red_target" = "xx11" && test "x$enable_client" = "xyes" ; then
 	if test "$os_linux" = yes; then
@@ -319,7 +319,7 @@
   AC_DEFINE([HAVE_XINERAMA], [], [Define if we have Xinerama])
   AC_SUBST(XINERAMA_CFLAGS)
   AC_SUBST(XINERAMA_LIBS)
-  SPICE_REQUIRES+=" xinerama"
+  SPICE_REQUIRES="$SPICE_REQUIRES xinerama"
 fi
 
 # Add parameter for (partial) static linkage of spice client.
