--- spice-0.8.1/server/Makefile.am.orig	2011-03-24 15:25:19.000000000 +0100
+++ spice-0.8.1/server/Makefile.am	2011-07-16 20:32:00.693757490 +0200
@@ -71,6 +71,7 @@
 	$(SLIRP_LIBS)				\
 	$(LIBRT)				\
 	$(Z_LIBS)				\
+	-lpthread				\
 	$(NULL)
 
 if SUPPORT_TUNNEL
--- spice-0.8.2/configure.ac.orig	2011-07-25 20:55:56.000000000 +0200
+++ spice-0.8.2/configure.ac	2011-07-25 20:57:18.293178544 +0200
@@ -237,6 +237,7 @@
 if test "x$enable_opengl" = "xyes"; then
    AC_CHECK_LIB(GL, glBlendFunc, GL_LIBS="$GL_LIBS -lGL", enable_opengl=no)
    AC_CHECK_LIB(GLU, gluSphere, GL_LIBS="$GL_LIBS -lGLU", enable_opengl=no)
+   AC_CHECK_LIB(X11, XOpenDisplay, GL_LIBS="$GL_LIBS -lX11", have_opengl=no)
    GL_CFLAGS="-DGL_GLEXT_PROTOTYPES -DUSE_OGL"
 
    if test "x$enable_opengl" = "xno"; then
