--- spice-0.12.6/spice-common/common/Makefile.am.orig	2015-08-14 10:54:53.000000000 +0200
+++ spice-0.12.6/spice-common/common/Makefile.am	2015-10-09 19:27:13.894734003 +0200
@@ -100,12 +100,6 @@
 	$(NULL)
 
 MARSHALLERS_DEPS =							\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/__init__.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/codegen.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/demarshal.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/marshal.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/ptypes.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/spice_parser.py	\
 	$(CODE_GENERATOR_BASEDIR)/spice_codegen.py			\
 	$(NULL)
 
--- spice-0.13.0/spice-common/tests/Makefile.am.orig	2016-01-13 13:27:47.000000000 +0100
+++ spice-0.13.0/spice-common/tests/Makefile.am	2016-04-03 08:33:52.601261670 +0200
@@ -27,12 +27,6 @@
 BUILT_SOURCES = $(TEST_MARSHALLERS)
 
 MARSHALLERS_DEPS =							\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/__init__.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/codegen.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/demarshal.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/marshal.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/ptypes.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/spice_parser.py	\
 	$(CODE_GENERATOR_BASEDIR)/spice_codegen.py			\
 	$(NULL)
 
